Tmain/interactive-mode.d/stdout-expected.txt

```diff
@@ -0,0 +1,46 @@
+identification message on startup
+=======================================
+{"name": "Universal Ctags", "version": "0.0.0"}
+
+error on invalid command
+=======================================
+{"name": "Universal Ctags", "version": "0.0.0"}
+{"error": "unknown command name", "fatal": true}
+
+error on missing arguments
+=======================================
+{"name": "Universal Ctags", "version": "0.0.0"}
+{"error": "invalid generate-tags request", "fatal": true}
+
+error on invalid file
+=======================================
+{"name": "Universal Ctags", "version": "0.0.0"}
+{"error": "cannot open input file \"test.foo\"", "warning": true, "errno": 2, "perror": "No such file or directory"}
+{"completed": "generate-tags"}
```

Comment:
Since we have `_type:"tag"` for tag outputs, maybe we should use something like `_type:"program"` for the welcome message on boot, and `_type:"error"` + `_type:"completed"` for the other tags. This should again make it simpler to integrate if a user can always expect a `_type` entry in returned objects.